### ISSUE TITLE
Actions, list of errors example

### ISSUE THREAD

**Issue by lnikell** [hasura/graphql-engine#4000]
I use the latest `hasura/graphql-engine:v1.2.0-beta.1`, based on the [documentation](https://hasura.io/docs/1.0/graphql/manual/actions/action-handlers.html) there is a way of sending a list of errors, but I did not find an example. Based on GraphQL specs the list of errors should look like this 
```
{
  "errors": [
    {
      "message": "Name for character with ID 1002 could not be fetched.",
      "locations": [ { "line": 6, "column": 7 } ],
      "path": [ "hero", "heroFriends", 1, "name" ],
      "extensions": {
        "code": "CAN_NOT_FETCH_BY_ID",
        "timestamp": "Fri Feb 9 14:33:09 UTC 2018"
      }
    }
  ]
}
```
But I was able to get successful response only by submitting single object:
```
{
  "message": "<error message>"
}
```

**Comment by rikinsk** [comment]
@lnikell This seems to be an issue in the docs. We'll update them.

Actually, the response JSON body is an object with a compulsory message field (text) and optional code field (text)

**Comment by lnikell** [comment]
@rikinsk thank you! so as a conclusion:
Line in the documentation https://hasura.io/docs/1.0/graphql/manual/actions/action-handlers.html
![image](https://user-images.githubusercontent.com/2697570/75771752-e4741f80-5d4a-11ea-8a23-13eca98979aa.png)
Should be replaced to:
`To return an error response, you must send back an error object. An error object looks like:`

**Comment by rikinsk** [comment]
Docs have been updated

**Comment by lnikell** [comment]
@rikinsk Thanks a lot!

**Comment by stefan-girlich** [comment]
See https://github.com/hasura/graphql-engine/discussions/5666 for the corresponding discussion.
tldr; A pull request with allows for custom error extensionsexists, but has unresolved merge conflicts since April 2020.
